vela-templates/definitions/internal/trait/storage.cue

```diff
@@ -279,7 +279,7 @@ template: {
 				envName:   string
 				secretKey: string
 			}
-			mountPath:   string
+			mountPath?:  string
```

Comment:
We should also change configmap's mountPath to optional since env can get value from configmap.

Reply:
> We should also change configmap's mountPath to optional since env can get value from configmap.

ok, I will add it

Reply:
> We should also change configmap's mountPath to optional since env can get value from configmap.

a new commit added cm's mountPath to be optional